lib/public_api.ts

```diff
@@ -1,14 +1,18 @@
 export { AccordionModule } from './src/accordion/accordion.module';
 export { BreadcrumbsModule } from './src/breadcrumbs/breadcrumbs.module';
 export {
-    AppSwitcherModule,
+    AppSwitcherModule
+} from './src/app-switcher/app-switcher.module';
+export {
+    AppSwitcherService,
+    MockAppSwitcherService
+} from './src/app-switcher/app-switcher.service';
+export {
     IAppSwitcherService,
     IAppSwitcherConfig,
     IDiscoveryApplication,
-    IDiscoveryRequest,
-    AppSwitcherService,
-    MockAppSwitcherService
-} from './src/app-switcher/index';
```

Comment:
Using direct imports instead of importing from barrel file.  This fixed the prod build issue.